src/utils/saml.ts

```diff
@@ -0,0 +1,21 @@
+import { DOMParser } from "xmldom";
+
+export interface ISamlResponse {
+  RelayState: string;
+  SAMLResponse: string;
```

Comment:
from the code below, this seems to be optional

Reply:
fixed in 1145b45